Doc/library/http.server.rst

```diff
@@ -392,8 +392,8 @@ provides three different variants:
       contents of the file are output. If the file's MIME type starts with
       ``text/`` the file is opened in text mode; otherwise binary mode is used.
 
-      For example usage, see the implementation of the :func:`test` function
-      invocation in the :mod:`http.server` module.
+      For example usage, see the implementation of the ``test`` function
+      invocation in :source:`Lib/http/server.py`.
```

Comment:
```suggestion
      in :source:`Lib/http/server.py`.
```
Sorry I missed this the first time, the word "invocation" doesn't add anything here.